imprint.html

```diff
@@ -81,6 +81,10 @@ <h3>Datenschutzerkl&auml;rung</h3>
 	</div>
 </div>
 
-</body>
+<!-- Matomo -->
+<script type="text/javascript" src="tracking.js"></script>
```

Comment:
I like that you're avoiding redundancy by putting the tracking code into a file. However, I'd add `async` and `defer` attributes, otherwise loading the page will be blocked by an additional HTTP request.